crates/router/src/configs/settings.rs

```diff
@@ -213,21 +213,47 @@ pub struct GenericLink {
     pub payout_link: GenericLinkEnvConfig,
 }
 
-#[derive(Debug, Deserialize, Clone, Default)]
+#[derive(Debug, Deserialize, Clone)]
 pub struct GenericLinkEnvConfig {
-    pub sdk_url: String,
+    pub sdk_url: url::Url,
     pub expiry: u32,
     pub ui_config: GenericLinkEnvUiConfig,
+    #[serde(deserialize_with = "deserialize_hashmap")]
     pub enabled_payment_methods: HashMap<enums::PaymentMethod, Vec<enums::PaymentMethodType>>,
 }
 
-#[derive(Debug, Deserialize, Clone, Default)]
+#[allow(clippy::panic)]
+impl Default for GenericLinkEnvConfig {
+    fn default() -> Self {
+        Self {
+            sdk_url: url::Url::parse("http://localhost:9050/HyperLoader.js")
+                .unwrap_or_else(|_| panic!("Failed to parse default SDK URL")),
```

Comment:
Use `expect()` instead.
And move the annotation `#[allow(clippy::expect_used)]` to just above the field, instead of on the entire implementation.

Same applies to the other `Default` implementation.